editor/src/consts.rs

```diff
@@ -57,5 +57,5 @@ pub const FILE_EXPORT_SUFFIX: &str = ".svg";
 pub const COLOR_ACCENT: Color = Color::from_unsafe(0x00 as f32 / 255., 0xA8 as f32 / 255., 0xFF as f32 / 255.);
 
 // Document
-pub const GRAPHITE_DOCUMENT_VERSION: &str = "0.0.4";
+pub const GRAPHITE_DOCUMENT_VERSION: &str = "0.0.5";
```

Comment:
Can you think of a better way we can enforce ourselves remembering to update this? Like create a unit test that imports a document which uses all the features of the document format and fails if it imports incorrectly, with a message reminding us to update that test plus this version number? (In the future with Graphene we'll have a more well-defined `.GDD` file format with a proper upgrade path. But for now this'll have to do.)

Reply:
I added a test. Is that what you are thinking. I didn't know the best place to but the sample file